pyproject.toml

```diff
@@ -138,6 +139,7 @@ changelog = "https://github.com/materialsproject/atomate2/blob/main/CHANGELOG.md
 
 [tool.setuptools.package-data]
 atomate2 = ["py.typed"]
+"atomate2.common.jobs" = ["*.json.gz"]
```

Comment:
presumably this line is here to include `src/atomate2/common/jobs/mp_avg_vol.json.gz` with the PyPI package? probably best to pass the full file path to not auto-include future `json.gz` file in this dir by accident. but more importantly, that file is 650KB and needs to be downloaded by everyone who installs `atomate2` if we merge the current config, whether they use MPMorph flows or not. up to @utf to make a decision here but i would vote against adding that file to the package. better to auto-download it when the user first needs it with a link where to get it if user doesn't have an internet connection.

finally, i looked into that file. looks like it's partially redundant. at least the `oxi_state` and `count` keys can likely be removed?

![Screenshot 2024-10-08 at 17 35 38](https://github.com/user-attachments/assets/e7e69c4e-6f54-4f5b-bb26-dae1992d4ca4)

Reply:
@janosh the oxidation states are I think ignored by default, but the counts are needed. If a given chemical environment doesn't exist in a database, the code looks through subspaces of that chemical environment and does a weighted average of structure volumes. The weighted average depends on the counts

I started rewriting this in parquet to jointly store them - might be better to put the parquet fiel on OpenData and pull down as needed

Reply:
OK so for clarity: for the ICSD data, the user can either query from oxidation-state dependent or agnostic data, since ICSD includes manually-assigned charges. I pulled the MP data from summary rather than bonds, so the oxidation state info there is ignored.

That's a slight incongruity that I can change if requested

The counts are needed for the same reason given above (the code will sample subspaces of the total chemical space to perform a weighted averaging of reference volumes/atom if the total chemical space is not present in the dataset)

The data is now pulled / cached from figshare

Stuck with JSON since the parquet speed bump is probably not super relevant here (reading basically once rather than reading many times, and the user has the option to manually specify a DataFrame as input, if generating many random structures)